design/open/lakefs-on-kv-testing-plan.md

```diff
@@ -0,0 +1,128 @@
+# lakeFS on KV - Testing Plan
+
+## Global Testing
+
+### Migration
+* Global tests to verify and benchmark the migration process
+* Need to verify that the system behavior is not affected by the move to the KV DB
+* [Question] - Do we need to dispose of these once migration to KV is globally completed?
```

Comment:
I think that migration code would be eventually deleted and so would the tests

Reply:
Sounds fair. Added a note

Reply:
@itaiad200 - If eventually we are getting rid of the migration code, how do users, that currently have lakeFS with Table DB installed, upgrade to the recent version? Are they forced to go through a Table->KV migration supporting version (i.e. at least 2 updates)?

Reply:
Yes. They are forced to do that. Eventually there would be a version where all lakeFS installations must go through.